pyccel/codegen/printing/fcode.py

```diff
@@ -658,8 +658,9 @@ def _print_NumpyEmpty(self, expr):
     def _print_NumpyNorm(self, expr):
         """Fortran print."""
 
-        if expr.dim:
-            rhs = 'Norm2({},{})'.format(self._print(expr.arg),self._print(expr.dim))
+        if expr.axis:
+            axis = '1' if expr.axis == 0 else self._print(expr.axis)
```

Comment:
The axis indexing begins at 0 in python and 1 in fortran. To get the same result you cannot just fix the first case. This will ensure that the code compiles and runs but it will not give the correct answer